### ISSUE TITLE
[fabric/forge-v2.2.1] Villagers do not display in the mod stats tab

### ISSUE THREAD

**Issue by yoyo3841** [TheCSMods/mc-better-stats#35]
**Describe the bug**
Villagers are not recorded in statistics. killing them is recorded in general mob kills, not on the mobs screen. Other stats are recorded traded with/talked to

**To Reproduce**
1. Open statistics screen
2. In general go to mob kills
3. In mobs go to # of kills
4. Kill a villager, mob kills will go up, # of kills will not
5. They won't show up on the mobs screen even with empty stats enabled

**Expected behavior**
Villagers existing in the mobs screen, and their kills being recorded too

**Comment by TheCSDev** [comment]
Hello. Thank you for reporting the bug.
I looked into it, and it appears I have made an attempt to hide all "non-living" entities such as boat, ender_pearl, snowball, fishing_bobber and so on from the BSS by filtering out all "miscellaneous" entities. However what I didn't realize is that Minecraft also considers villagers as "miscellaneous" entities as well, hence why villagers disappeared from the BSS. I will fix that issue as soon as I can.
